webserver_test.py

```diff
@@ -0,0 +1,43 @@
+import re
+from subprocess import Popen
+from subprocess import PIPE
+import sys
+
+# The expected output from curl as a regular expression
+curl_output_expected = ("GET / HTTP/1\\.0\r\n"
+                        "Host: localhost:\\d+\r\n"
+                        "User-Agent: curl/\\d+\\.\\d+\\.\\d+\r\n"
+                        "Accept: \\*/\\*\r\n")
```

Comment:
If my reg expression knowledge is correct, this line can be stronger. I assume we would only want to accept alphabetical characters here so something along [a-zA-Z]* should be good